python/ray/tune/integration/keras.py

```diff
@@ -65,6 +73,7 @@ def _handle(self, logs: Dict, when: str = None):
         tune.report(**report_dict)
 
 
+@Deprecated(message=_DEPRECATION_MESSAGE)
 class _TuneCheckpointCallback(TuneCallback):
     """Keras checkpoint callback
```

Comment:
Any reason for annotating this nonpublic class?

Reply:
No reason really, felt like it's nice to tag for deletion later.